isic_archive/models/image.py

```diff
@@ -29,16 +28,26 @@ def initialize(self):
 
         self.prefixSearchFields = ['lowerName', 'name']
 
+    @staticmethod
+    def generateIsicID():
```

Comment:
Do we have the infrastructure to easily add a test, to assert that this returns something like `ISIC_\d{7}`? If this will take more than 10 minutes, I don't think it's worth it.